### DISCUSSION TITLE
namespace argocd is not permitted in project 'my-project'

### DISCUSSION THREAD

**Discussion by mmanirmd** [argoproj/argo-cd#12458] [General]
When using custom AppProject in App Of Apps patern, the child app is giving  "namespace argocd is not permitted in project 'my-project'" error message but if I also include argocd namespace in appproject's destination namespace, that fix the issue. 
Any idea how can I fix the issue? Thanks in advance.

**Comment by jannfis** [comment]
Which of those apps do belong to the `my-project` AppProject, the parent app or the child apps?
